scripts/deploy_hcloud.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+set -e
+git config --global push.default simple # we only want to push one branch — master
+git fetch --unshallow origin
+ssh-keyscan -H $IP >> ~/.ssh/known_hosts
+# add repo on vps as a repote
+git remote add production ssh://$USER@$IP/home/$USER/susi-server-git
+# push updates
+git push -f production HEAD:master
+# build and start susi server
+ssh $USER@$IP <<EOF
+  cd susi-server-git
+  bin/stop.sh
+  git submodule update --recursive --remote
+  git submodule update --init --recursive
+  ./gradlew build -x test
+EOF
+
+ssh $USER@$IP <<EOF
```

Comment:
Is there a reason you are using two separate ssh processes, and not all in one go?

Reply:
@norbusan Yes, When the Gradle build is completed it is closing the ssh session. Because of that, we have to start a new session to start the server.

Reply:
Indeed, strange ... just tried it after your comment and it actually does that - which is something I wouldn't expect.